src/Modules/OrchardCore.Commerce/Views/Checkout.cshtml

```diff
@@ -11,7 +13,7 @@
 
 <section id="checkout">
     <div class="payment-form_container">
-        <form id="payment-form" class="payment-form">
+        <form asp-action="@nameof(PaymentController.CheckoutWithoutPayment)" asp-route-shoppingCartId="@Model.ShoppingCartId" id="payment-form" class="payment-form">
```

Comment:
When the Stripe payment form is rendered, this form is not submitted, so it doesn't interfere with payment checkouts.